icons.go

```diff
@@ -12,8 +12,10 @@ type AppData interface {
 
 // ApplicationProvider describes a type that can locate icons and applications for the current system
 type ApplicationProvider interface {
+	AllApps() []AppData
+	AvailableThemes() []string
```

Comment:
Why the difference between AllApps but AvailableThemes? it feels like this should be consistent.